WordPress/wp-feature-notifications
# Feature/notification Design implementation (follow-up of #66)

_This is a followup of #66 PR_

Following the @jonathanbossenger [directions](https://github.com/WordPress/wp-notify/pull/66#issuecomment-1123866478) I've merged the #66 html templates into an installable plugin that shows the current design so that anyone installing it has a preview of the design. 

Please however consider it as a WIP since some sections actually are merely sketched (for example the sidebar part is fully hardcoded). In addition there are some fix needed that @bacoords has [pointed out](https://wordpress.slack.com/archives/C2K1C71FE/p1652881640629739?thread_ts=1652700859.072489&cid=C2K1C71FE)

**How to test:**
- clone this branch, cd the plugin directory and `wp-env start` (you must have docker running)
- if the plugin works you should see at the top of the dashboard two buttons and a welcome notification
- Open devtools then into the console try with:
`addNotify({title: 'My title', message:'my message', image:'https://source.unsplash.com/random/400×400'})` or
`removeNotify(0)` or `clearNotifies()`
- Check for Settings -> Notification (options-general.php?page=wp-notify)
